### PR TITLE
CPlusPlus.yaml: Adds datatypes

### PR DESCRIPTION
**Pull request by SerekKiri**
CPlusPlus.yaml --> Added data types for cpp

### CODE REVIEW

File: data/Language/CPlusPlus.yaml [hunk 1]
```diff
@@ -119,3 +119,12 @@ keywords:
   - '#elif'
   - '#line'
   - '#pragma'
+datatypes:
+  - bool_cpp
+  -	int_cpp
```
Comment (jayvdb) [inline data/Language/CPlusPlus.yaml]: what is going on here ... :P

Comment (SerekKiri) [reply data/Language/CPlusPlus.yaml]: Fixed

File: data/Delimiter/cpp_char.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+type: string
```
Comment (jayvdb) [inline data/Delimiter/cpp_char.yaml]: remove this , and use `c_char.yaml` - they are the same delimiter definition , I believe.

If someone disagrees, they can split them into two with proof they are different.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coAST/pull/38/commits/46d4890b605b6d63a0b0ab0cc6e920de68744c3a

**Comment by jayvdb** [comment]
ack 46d4890

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being fastforwarded automatically. Please **DO NOT** push while fastforward is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated fastforward with [GitMate.io](https://gitmate.io) was successful! :tada:

**Pull request merged**